src/app/pages/adoption-form/book-info/book-info.js

```diff
@@ -0,0 +1,74 @@
+import BaseView from '~/helpers/backbone/view';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './book-info.hbs';
+
+function requireOne(container) {
+    let checkboxes = container.querySelectorAll('input[type=checkbox]'),
+        checkCount = 0,
+        setClass = function () {
+            if (checkCount > 0) {
+                container.classList.remove('invalid');
+            } else {
+                container.classList.add('invalid');
+            }
+        },
+        onChange = function (e) {
+            if (e.target.checked) {
+                checkCount += 1;
+            } else {
+                checkCount -= 1;
+            }
+            setClass();
+        };
+
+    for (let i=0; i<checkboxes.length; ++i) {
+        let el = checkboxes[i];
+
+        el.addEventListener('change', onChange);
+    }
+    setClass();
+}
+
+@props({
+    template: template
+})
+export default class BookInfoView extends BaseView {
+    constructor(prefix) {
+        super();
+        this.templateHelpers = {
+            prefix: prefix
+        };
+    }
+    onRender() {
+        let checkboxGroup = this.el.querySelector('.require-one');
+
+        requireOne(checkboxGroup);
+
+        let noneBox = this.el.querySelector('input[type="checkbox"][value="None"]');
+
+        noneBox.addEventListener('change', () => {
```

Comment:
Adding a new event listener every time to element is rendered will create a memory leak, and may cause some unexpected behavior as well.